.semaphore/semaphore.yml

```diff
@@ -65,15 +65,20 @@ blocks:
       prologue:
         commands:
           - checkout && cd ~/git/flowcrypt-ios/ && cache restore && make dependencies && cache store
-          - sem-version node 16 && mkdir .custom-npm && cd .custom-npm && npm install npm@8.1.2 && cd ..
+          - 'wget https://github.com/FlowCrypt/flowcrypt-ios/files/8811587/custom-npm.zip -P ~/git/flowcrypt-ios'
```

Comment:
Not sure why we used custom-npm (v8.1.2) before.
Anyway, now root project contains pacakage.json and previous method of `npm install npm@8.1.2` won't work (need to do npm init first).
To achieve this, I had to upload & use custom-npm folder which contains npm v8.1.2

Reply:
Use git blame to understand historical context. It was introduced here

https://github.com/FlowCrypt/flowcrypt-ios/blame/14b56af9f738f1b21e6118f3fc424fc461164908/.semaphore/semaphore.yml

https://github.com/FlowCrypt/flowcrypt-ios/issues/506

https://github.com/FlowCrypt/flowcrypt-ios/pull/512

Looks to be there for building the Core folder. It may have only built properly on some npm versions and not others.

<img width="934" alt="image" src="https://user-images.githubusercontent.com/6306961/171354469-f8753d0c-c855-4198-b2c8-3dedac924a8a.png">


Why not try the latest npm (a specific version) and since you are using it for everything, why not update the global one? The zipped version looks even more hacky than the previous setup.